changelog/_unreleased/2021-02-14-fix-sending-emails-did-not-use-language-chain-fallback-for-header-and-footer-mail-templates.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Fix sending emails did not use language chain fallback for header and footer mail templates
+issue: NEXT-13729
+---
+# Core
+* Upon sending emails sse language chain fallback when retrieving header and footer mail template content.
```

Comment:
Please fix your changelog to use our given keywords at the beginning. See: https://github.com/shopware/platform/blob/master/adr/2020-08-03-Implement-New-Changelog.md

Reply:
@bneumann97 Done. I hope now it is as you expect it.